Bump 2.0.0

evonreis opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

erikvonreis committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.11.19.38.21

erikvonreis committed:
bump to 2.0.0
also add windows build
also drop all patches, each of which has been merged into the source

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

erikvonreis committed:
Fix some build issues

add prefix/include and prefis/Library/include to includes dirs
add dl lib back in for unix builds

erikvonreis committed:
add missing patches

erikvonreis committed:
Fix ctest run for windows

erikvonreis committed:
don't test on cross-builds

erikvonreis committed:
Fix tests for windows

erikvonreis committed:
Further fixes to windows tests
